charms/build-and-release-all-parallel.groovy

```diff
@@ -0,0 +1,50 @@
+stage('charms') {
+    def jobs = [:]
+    
+    charms_to_build = ['easyrsa', 'etcd', 'flannel', 'kubeapi-load-balancer', 
+    'kubernetes-master', 'kubernetes-worker', 'kubernetes-e2e', 'calico', 'canal'
+    ]
+    
+    charms_to_build.each { charm_name ->
+        jobs[charm_name] = {
+            stage(charm_name) {
+                build job: 'build-and-release-' + charm_name, parameters: [
+                    booleanParam(name: 'RUN_TESTS', value: true),
```

Comment:
Can we make the RUN_TESTS an environment variable? There are cases where we want to push charms and bundles to edge even if there are tests failing.